committee/2017-03-03/minutes.md

```diff
@@ -0,0 +1,127 @@
+#Minutes
+
+##Aberystwyth Computer Science Society
+
+##Committee Meeting 1
+
+##05/11/15
+
+
+####Present:
+
+- Lizzie Stone (eas12) - President
+- Cloud Coppola (clc20) - Vice-President
+- Jack Webb (jaw84) - Treasurer
+- Samuel Claxton - Acting Secretary/Social Secretary
+- Paul Swain (pas28) - Social Secretary
+- Aidan Hodgen (aih16) - Publicity Officer
+
+
+###Apologies:
+
+- None
+
+####Item 1 - Discussion of changing committee channel to private:
+  
+  - Committee ruled against, stating transparancy as important
+  - Outlined that Email is to be used for sensitive information
+    - In such instances, society email should be Carbon Copied in (SCTY14@aber.ac.uk)
+
+
+####Item 2 - Role of Publicity Officer:
+
+  - Committee discussed the role of the publicity officer and decreed the following:
+    - The publicity officer is in charge of the societies social media image spanning facebook and twitter
+    - The publicity officer is in charge of promotional materials for fresher fairs and other such events
+
+####Item 3 - Discussion of communications of CSTF:
+
+  - The Committee discussed how best to communicate with the Cyber Security Tasf Force, and decided on the following points
+    - The society needs to outline a set of well defined tandards and morals for documents, files on git etc.
+    - A comprehensive, specific and unambigious treaty should be drawn up with the Cyber Security Task Force detailing our 
+    relationship and outling where we are heading
+
+####Item 4 - CSTF Legal issues:
+
+  - The Committee discussed the potential legal issues concerned with the Cyber Security Task Force and ruled the following:
+    - CSTF is to provide full details on projects such that potential legal issues can be identified
+    - The possibility of legal forms, although the committee accepts it may need guidance on this
+
+
+
+####Item 5 - Sponsorships:
+
+  - Carried Forward (Jack Webb (jaw84))
+
+####Item 6 - Twitter:
+
+  - For now the banner from facebook has been transfered accross to twitter
+    - Temporary solution until new promotional matirials are produced
+
+####Item 7 - Changes to the AberCompSoc Image:
+
+  - The Committee discussed changes to the image of the branding and concluded the following
+    - Keep the logo black and white
+    - Changes to banner (e.g. change layout and colours)
+  - Carried Forward
+
+####Item 8 - Society Font Change:
+
+  - The Committee discussed changes to the font of the branding and concluded the following
+    - Some change is needed as it is not openable on all systems
```

Comment:
The minutes only reflect what was covered in the meeting, this can however be added onto the agenda for the next society meeting